### DISCUSSION TITLE
Install worlds?

### DISCUSSION THREAD

**Discussion by revvr** [itzg/docker-minecraft-server#2228] [General]
I run a Bedrock Minecraft server for my son on a Docker container in my NAS. For his bday, he got a $20 gift card which I loaded into his account. He now has 3,500 coins.

He plays exclusively on this server with friends.

If he uses those coins to buy worlds, etc. from the Minecraft marketplace, will I be able to load that into the server? If so, how?

I'm hoping that he gets stuff that other players can see/use (textures, worlds, skins, etc.) Are there any of these that won't work at all with this docker minecraft server?

Thanks!!

**Comment by itzg** [comment]
It sounds like it may not be possible

https://feedback.minecraft.net/hc/en-us/community/posts/360062513872-Marketplace-content-on-Bedrock-Dedicated-Server

**Comment by revvr** [comment]
Thank you. A real bummer and head scratcher for sure. I can't believe it is a problem. I guess I'll wait.

Hey, and thank you so much for maintaining this. I can't even begin to tell you how many hours my son and his friends spend playing Minecraft on my NAS's docker container. Thanks again!!!

**Reply by itzg** [reply]
I know, right? It's a bummer that the money spent can't be translated over. I get it though, it's big business for Mojang/Microsoft with the marketplace, so they'll hold a death grip on it.
